project.godot

```diff
@@ -19,3 +19,34 @@ config/icon="res://icon.svg"
 
 version_control/plugin_name="GitPlugin"
 version_control/autoload_on_startup=true
+
+[input]
+
+move_up={
```

Comment:
Humm, use more generic names like just `up`.
For example, you're in the menu, you probably will want to use the arrows or the WASD to move in the menu.
What you think? :)